### DISCUSSION TITLE
Dory SSL

### DISCUSSION THREAD

**Discussion by 20Rishabh20** [dspeterson/dory#29] [General]
Hi,

Need to implement TLS connection with Kafka using dory.
How can we achieve this?

**Comment by dspeterson** [comment]
Hi 20Rishabh20,

If you are interested in adding TLS support to dory, that would be awesome!  To get started, see my reply to your comment under "Issues" for some pointers to places in the code to look at.  See also https://github.com/dspeterson/dory/blob/master/doc/dev_info.md for general info for developers.  If you have further questions, please don't hesitate to ping me.

Thanks,
Dave
